src/main/java/com/bmc/truesight/saas/meter/client/command/AddEvents.java

```diff
@@ -86,6 +87,7 @@ private static String toEventString(Event event) {
 
         event.at().ifPresent(at -> fields.add(format(AT_ID, at)));
         event.ad().ifPresent(ad -> fields.add(format(AD_ID, ad)));
+        event.properties().ifPresent(pMap -> fields.add(format(PROPERTIES_ID, pMap.toString().substring(1, pMap.toString().length() - 1))));
```

Comment:
You should never rely on toString() for anything other than logging. It's probably safe here but it's a bad idea in general.